From 264cf17882d975b960d18496541b4bada1a78383 Mon Sep 17 00:00:00 2001
From: Liu Ying <b17645@freescale.com>
Date: Fri, 8 Jan 2010 15:44:13 -0500
Subject: [PATCH 143/505] ENGR00119710 MXC V4L2 output:Support YUYV input
 pixel format

Support YUYV input pixel format for mxc V4L2 output unit test.

Signed-off-by: Liu Ying <b17645@freescale.com>
---
 test/mxc_v4l2_test/mxc_v4l2_output.c | 3 ++-
 1 file changed, 2 insertions(+), 1 deletion(-)

diff --git a/test/mxc_v4l2_test/mxc_v4l2_output.c b/test/mxc_v4l2_test/mxc_v4l2_output.c
index 8848bba..d7e6b26 100644
--- a/test/mxc_v4l2_test/mxc_v4l2_output.c
+++ b/test/mxc_v4l2_test/mxc_v4l2_output.c
@@ -1,5 +1,5 @@
 /*
- * Copyright 2004-2009 Freescale Semiconductor, Inc. All rights reserved.
+ * Copyright 2004-2010 Freescale Semiconductor, Inc. All rights reserved.
  */
 
 /*
@@ -494,6 +494,7 @@ int process_cmdline(int argc, char **argv)
                              (g_in_fmt != V4L2_PIX_FMT_RGB565) &&
                              (g_in_fmt != 'PMBW') &&
                              (g_in_fmt != V4L2_PIX_FMT_UYVY) &&
+			     (g_in_fmt != V4L2_PIX_FMT_YUYV) &&
 			     (g_in_fmt != V4L2_PIX_FMT_YUV422P) &&
                              (g_in_fmt != V4L2_PIX_FMT_YUV420) &&
                              (g_in_fmt != V4L2_PIX_FMT_NV12) )
-- 
1.8.0

